lib/pina/models/sales_order.rb

```diff
@@ -0,0 +1,98 @@
+module Pina
+  module Models
+    class SalesOrder
+      include Virtus.model
+
+      attribute :created_at
+      attribute :updated_at
+      attribute :creator
+      attribute :creator_id
+      attribute :modifier
+      attribute :modifier_id
+      attribute :accounting_address
+      attribute :accounting_address_id
+      attribute :buyer_address
+      attribute :buyer_address_id
+      attribute :document_recipient_address
+      attribute :document_recipient_address_id
+      attribute :delivery_address
+      attribute :delivery_address_id
+      attribute :contact
+      attribute :currency
+      attribute :currency_id
+      attribute :date_of_receipt
+      attribute :delivery_method
+      attribute :department
+      attribute :department_id
+      attribute :percent_discount
+      attribute :external_id
+      attribute :fullfilled_on
+      attribute :group_id
+      attribute :invoicing_method
+      attribute :status
+      attribute :order
```

Comment:
jj pocital sem s tim, neni tam POST ne PATCH, je to jen GET index/show, ty parametry mam z odpovedi + podle sales_invoice parsovani parametru, takze je to ok ;)